app/lib/features/link_room/pages/link_room_page.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:acter/common/extensions/options.dart';
```

Comment:
the core structure of these has remained the same, but they appears a new files because I moved the content into its own feature and split it up into several distinct widgets that I can also easier test on their own.